packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -23,6 +19,8 @@ import {
 import {
   ReactNativeViewConfigRegistry,
   deepFreezeAndThrowOnMutationInDev,
+  createPublicInstance,
+  type PublicInstance as FabricPublicInstance,
```

Comment:
Given that we're trying to remove codenames from the repo. Should this be `ReactNativePublicInstance`?